src/current/v24.1/create-function.md

```diff
@@ -31,7 +31,7 @@ If you grant `EXECUTE` privilege as a default privilege at the database level, n
 Parameter | Description
 ----------|------------
 `routine_create_name` | The name of the function.
-`routine_param` | A comma-separated list of function parameters.
+`routine_param` | A comma-separated list of function parameters, specifying the mode, name, and type.
```

Comment:
Does this mean that you set `routine_param` to a series of mode:name:type groupings? Is there an example anywhere?